tools/pythonpkg/duckdb/__init__.py

```diff
@@ -0,0 +1,325 @@
+    "append",
+    "array_type",
+    "arrow",
+    "begin",
+    "close",
+    "commit",
+    "create_function",
+    "cursor",
+    "decimal_type",
+    "description",
+    "df",
+    "dtype",
+    "duplicate",
+    "enum_type",
+    "execute",
+    "executemany",
+    "fetch_arrow_table",
+    "fetch_df",
+    "fetch_df_chunk",
+    "fetch_record_batch",
+    "fetchall",
+    "fetchdf",
+    "fetchmany",
+    "fetchnumpy",
+    "fetchone",
+    "filesystem_is_registered",
+    "from_arrow",
+    "from_csv_auto",
+    "from_df",
+    "from_parquet",
+    "from_query",
+    "from_substrait",
+    "from_substrait_json",
+    "get_substrait",
+    "get_substrait_json",
+    "get_table_names",
+    "install_extension",
+    "interrupt",
+    "list_filesystems",
+    "list_type",
+    "load_extension",
+    "map_type",
+    "pl",
+    "query",
+    "read_csv",
+    "read_json",
+    "read_parquet",
+    "register",
+    "register_filesystem",
+    "remove_function",
+    "rollback",
+    "row_type",
+    "sql",
+    "sqltype",
+    "string_type",
+    "struct_type",
+    "table",
+    "table_function",
+    "tf",
+    "torch",
+    "type",
+    "union_type",
+    "unregister",
+    "unregister_filesystem",
+    "values",
+    "view"
+])
+
+# Exceptions
+from .duckdb import (
+    Error,
+    DataError,
+    CastException,
+    ConversionException,
+    OutOfRangeException,
+    TypeMismatchException,
+    ValueOutOfRangeException,
+    FatalException,
+    IntegrityError,
+    ConstraintException,
+    InternalError,
+    InternalException,
+    InterruptException,
+    NotSupportedError,
+    NotImplementedException,
+    OperationalError,
+    ConnectionException,
+    IOException,
+    HTTPException,
+    OutOfMemoryException,
+    SerializationException,
+    TransactionException,
+    PermissionException,
+    ProgrammingError,
+    BinderException,
+    CatalogException,
+    InvalidInputException,
+    InvalidTypeException,
+    ParserException,
+    SyntaxException,
+    SequenceException,
+    StandardException,
+    Warning
+)
+_exported_symbols.extend([
+    "Error",
+    "DataError",
+    "CastException",
+    "ConversionException",
+    "OutOfRangeException",
+    "TypeMismatchException",
+    "ValueOutOfRangeException",
+    "FatalException",
+    "IntegrityError",
+    "ConstraintException",
+    "InternalError",
+    "InternalException",
+    "InterruptException",
+    "NotSupportedError",
+    "NotImplementedException",
+    "OperationalError",
+    "ConnectionException",
+    "IOException",
+    "HTTPException",
+    "OutOfMemoryException",
+    "SerializationException",
+    "TransactionException",
+    "PermissionException",
+    "ProgrammingError",
+    "BinderException",
+    "CatalogException",
+    "InvalidInputException",
+    "InvalidTypeException",
+    "ParserException",
+    "SyntaxException",
+    "SequenceException",
+    "StandardException",
+    "Warning"
+])
+
+__all__ = _exported_symbols
```

Comment:
Is there a reason you're not calling it `__all__` to begin with?

Reply:
I got warnings/errors that modifications on `__all__` are not supported, so I'm creating the list first and only then assigning it.

I'd like to replace this with a solution that doesn't involve repetition between importing the symbols and adding them to list, but couldn't quickly find a way to do so.